credentials/pom.xml

```diff
@@ -54,6 +54,10 @@
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>com.google.http-client</groupId>
+          <artifactId>google-http-client</artifactId>
```

Comment:
Is this dependency needed? With the wildcard import, it's hard to see how it's being used.

Reply:
added it for Precondition, but since removed as per discussion below, that is overkill